Refresh.GameServer/Database/Activity/GameDatabaseContext.ActivityWrite.cs

```diff
@@ -18,7 +18,7 @@ public Event CreateLevelUploadEvent(GameUser userFrom, GameLevel level)
         {
             EventType = EventType.LevelUpload,
             StoredDataType = EventDataType.Level,
-            Timestamp = this._time.TimestampMilliseconds,
+            Timestamp = this._time.Now,
```

Comment:
This isnt a UTC timestamp, so when its stored in realm it will drop the offset. `SystemDateTimeProvider` should be modified to return a UTC timestamp for `.Now`, see [here](https://github.com/LittleBigRefresh/Refresh/blob/3b6d939baae5700aee3eb636510957eb2a3a27dc/Refresh.GameServer/Time/SystemDateTimeProvider.cs#L10)

Reply:
...This has broken this entire time?

Reply:
seemingly so